setup.py

```diff
@@ -13,15 +14,17 @@
 
 setup(
     name='charm-benchmark',
-    version='0.0.1',
+    version='0.0.2.2',
     description='Library to aid in the creation of benchmark actions in Juju',
     install_requires=install_requires,
-    url="https://launchpad.net/~cabs-team",
-    packages=['charmbenchmark'],
+    url="https://github.com/juju-solutions/charm-benchmark",
+    packages=['charmbenchmark', 'charmbenchmark.cli'],
```

Comment:
replace this with `packages=find_packages()` and import `[find_packages](https://pythonhosted.org/setuptools/setuptools.html#using-find-packages)` from `setuptools` at the top